tests/migrations/test_optimizer.py

```diff
@@ -631,3 +632,34 @@ def test_optimize_through_fields(self):
                 migrations.CreateModel("Bar", [("width", models.IntegerField())]),
             ],
         )
+
+    def test_optimize_elidable_operation(self):
+        self.assertOptimizesTo(
+            [
+                operations.special.ElidableOperation(elidable=True),
```

Comment:
When you drop this operation the test fails with the error shown below. This is due to the fact that the `CreateModel` operation as well as any operations don't know anything about the `elidable` flag. Thus you'd either add

```python
elif isinstance(operation, ElidableOperation) and operation.elidable:
    return [self]
```

or abuse `operation.references_model()` and `operation.references_field()` or add something to check for elidability.

```python
======================================================================
FAIL: test_optimize_elidable_operation (migrations.test_optimizer.OptimizerTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/markus/Coding/django/tests/migrations/test_optimizer.py", line 661, in test_optimize_elidable_operation
    migrations.CreateModel("Bar", [("size", models.IntegerField())]),
  File "/home/markus/Coding/django/tests/migrations/test_optimizer.py", line 27, in assertOptimizesTo
    self.assertEqual(expected, result)
AssertionError: Lists differ: ["('ElidableOperation', (), {'elidable': Fals[251 chars]]})"] != ["('CreateModel', [], {'name': 'Foo', 'fields[201 chars]]})"]

First differing element 0:
('ElidableOperation', (), {'elidable': False})
('CreateModel', [], {'name': 'Foo', 'fields': [('name', <django.db.models.fields.CharField>)]})

First list contains 1 additional elements.
First extra element 3:
('CreateModel', [], {'name': 'Bar', 'fields': [('size', <django.db.models.fields.IntegerField>)]})

- ["('ElidableOperation', (), {'elidable': False})",
-  "('CreateModel', [], {'name': 'Foo', 'fields': [('name', "
? ^

+ ["('CreateModel', [], {'name': 'Foo', 'fields': [('name', "
? ^

   '<django.db.models.fields.CharField>)]})',
   "('ElidableOperation', (), {'elidable': False})",
   "('CreateModel', [], {'name': 'Bar', 'fields': [('size', "
   '<django.db.models.fields.IntegerField>)]})']
```